service/src/main/java/bio/terra/workspace/service/workspace/GcpCloudSyncRoleMapping.java

```diff
@@ -92,7 +118,40 @@ public class GcpCloudSyncRoleMapping {
               "notebooks.operations.delete",
               "notebooks.operations.get",
               "notebooks.operations.list",
-              "serviceusage.services.use")
+              "serviceusage.services.use",
+              "dataproc.jobs.create",
+              "dataproc.jobs.update",
+              "dataproc.jobs.delete",
```

Comment:
double checking - does giving this at the project level allow users to delete jobs in any dataproc cluster (which would include private ones not assigned to them), or do they also need cluster-level permission?